setup.py

```diff
@@ -98,7 +98,7 @@ def _compose_list(values, prefix=None):
     test_suite = "tests.unittests",
     zip_safe = True,
     setup_requires=["pytest-runner"],
-    tests_require=["pytest"],
+    tests_require=["pytest==7.0.0"],
```

Comment:
let's undo this pin now that we're pinning the problematic python36/7 via requirements.txt in the CI